css/style.css

```diff
@@ -489,8 +642,12 @@ body {
     max-width: 305px;
   }
 
+.test__row{
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/chendler_tesla/pullRequest?prid=5991762)